In [1]:
name = "Leon"
hourly_wage = 20.0
hours = 32

weekly_pay = hourly_wage * hours

print(f"{name} earns {weekly_pay} per week.")

Leon earns 640.0 per week.


In [2]:
temps = [61, 65, 72, 58, 70, 74, 68]

warm_days = 0

for temp in temps:
    if temp > 65:
        warm_days = warm_days + 1

print(warm_days)

4


In [3]:
def to_celsius(f):
    return (f - 32) * 5 / 9

print(round(to_celsius(temps[0]), 1))

16.1


In [4]:
movie = {
    "title": "Interstellar",
    "year": 2014,
    "rating": 8.7
}

for key, value in movie.items():
    print(key, ":", value)

title : Interstellar
year : 2014
rating : 8.7


In [5]:
import pandas as pd

df = pd.read_csv("chickwts.csv")

df.head()

,Unnamed: 0,weight,feed
0,1,179,horsebean
1,2,160,horsebean
2,3,136,horsebean
3,4,227,horsebean
4,5,217,horsebean


In [6]:
df.shape

(71, 3)

In [7]:
df.describe()

,Unnamed: 0,weight
count,71.000000,71.000000
mean,36.000000,261.309859
std,20.639767,78.073700
min,1.000000,108.000000
25%,18.500000,204.500000
50%,36.000000,258.000000
75%,53.500000,323.500000
max,71.000000,423.000000


In [8]:
import os
os.listdir()

['.ipynb_checkpoints', 'chickwts.csv', 'module2_lab.ipynb']

In [9]:
df = pd.read_csv("chickwts.csv", index_col=0)

In [10]:
df.head()

,weight,feed
1,179,horsebean
2,160,horsebean
3,136,horsebean
4,227,horsebean
5,217,horsebean


In [11]:
df.shape

(71, 2)

In [12]:
df.describe()

,weight
count,71.000000
mean,261.309859
std,78.073700
min,108.000000
25%,204.500000
50%,258.000000
75%,323.500000
max,423.000000


Basically, what the output is telling me is that the average chick weighed at about 260. Within the ranges, the weights go from anywhere from 108 to 423, so its safe to say there was a big spread. What stood how the most however/the main point, how much the weight varied between the lightest and heaviest chick.